scripts/ecs/appsignals/setup-ecs-demo.sh

```diff
@@ -0,0 +1,1124 @@
+  service_discovery_id=$(aws servicediscovery create-service \
+      --name config-server \
+      --dns-config "NamespaceId=$namespace_id,RoutingPolicy=WEIGHTED,DnsRecords=[{Type=A,TTL=300}]" \
+      --health-check-custom-config FailureThreshold=2 \
+      --query "Service.Id" --output text)
+
+  registryArn=$(aws servicediscovery get-service \
+    --id $service_discovery_id \
+    --query 'Service.Arn' --output text)
+
+  aws ecs create-service \
+      --service-name config-server \
+      --task-definition config-server \
+      --service-registries registryArn=$registryArn \
+      --desired-count 1 \
+      --launch-type FARGATE \
+      --network-configuration "awsvpcConfiguration={subnets=[$(echo $SUBNET_IDS | tr -s ' ' ',')],securityGroups=[$SECURITY_GROUP_ID],assignPublicIp=ENABLED}" \
+      --cluster $CLUSTER > /dev/null
+
+  echo "Waiting for the config server to be accessible..."
+  sleep 180
+  echo "Config server is now accessible!"
+
+}
+
+function run_discovery_server() {
+  sed -i '' "s|\"discovery-server-image\"|\"${ECR_IMAGE_PREFIX}/springcommunity/spring-petclinic-discovery-server\"|" ./sample-app/task-definitions/spring-petclinic-discovery-server.json
+  sed -i '' "s|us-west-2|${REGION}|g" ./sample-app/task-definitions/spring-petclinic-discovery-server.json
+  sed -i '' "s|000111222333|${ACCOUNT_ID}|g" ./sample-app/task-definitions/spring-petclinic-discovery-server.json
+
+  aws ecs register-task-definition --cli-input-json file://sample-app/task-definitions/spring-petclinic-discovery-server.json > /dev/null
+
+  namespace_id=$(aws servicediscovery list-namespaces --query "Namespaces[?Name=='ecs-petclinic'].Id" --output text)
+  service_discovery_id=$(aws servicediscovery create-service \
+      --name discovery-server \
+      --dns-config "NamespaceId=$namespace_id,RoutingPolicy=WEIGHTED,DnsRecords=[{Type=A,TTL=300}]" \
+      --health-check-custom-config FailureThreshold=2 \
+      --query "Service.Id" --output text)
+
+  registryArn=$(aws servicediscovery get-service \
+    --id $service_discovery_id \
+    --query 'Service.Arn' --output text)
+
+  aws ecs create-service \
+      --service-name discovery-server \
+      --task-definition discovery-server \
+      --service-registries registryArn=$registryArn \
+      --desired-count 1 \
+      --launch-type FARGATE \
+      --network-configuration "awsvpcConfiguration={subnets=[$(echo $SUBNET_IDS | tr -s ' ' ',')],securityGroups=[$SECURITY_GROUP_ID],assignPublicIp=ENABLED}" \
+      --cluster $CLUSTER > /dev/null
+
+  echo "Waiting for the discovery server to be accessible..."
+  sleep 180
+
+  echo "discovery server is now accessible!"
+
+}
+
+function run_admin_server() {
+  adot_java_image_tag=$(curl -s -I -L 'https://github.com/aws-observability/aws-otel-java-instrumentation/releases/latest' | grep -i Location | awk -F'/tag/' '{print $2}' | tr -d '\r')
```

Comment:
IMO, admin-server, discovery-server and config-server don't need auto-instrumentation. Let me know if you think differently.